docs/MO_DG/prepare_model/convert_model/Cutting_Model.md

```diff
@@ -1,15 +1,15 @@
 # Cutting Off Parts of a Model  {#openvino_docs_MO_DG_prepare_model_convert_model_Cutting_Model}
 
-Sometimes some parts of a model must be removed while the Model Optimizer is converting models to the Intermediate Representation. This chapter describes methods of doing cutting off parts of a model using Model Optimizer command-line options. Model cutting applies mostly to TensorFlow\* models, but is also useful for other frameworks. In this chapter, TensorFlow examples are used for illustration.
+Sometimes, some parts of a model must be removed while the Model Optimizer converts models to the Intermediate Representation. This chapter describes methods of cutting off parts of a model using Model Optimizer command-line options. Model cutting applies mostly to TensorFlow models, but it is also useful for other frameworks. TensorFlow examples are used in this chapter for illustrative purposes.
```

Comment:
```suggestion
Sometimes, it is necessary to remove parts of a model when converting it to OpenVINO IR. This chapter describes how to do it, using Model Optimizer command-line options. Model cutting applies mostly to TensorFlow models, which is why TensorFlow will be used in this chapter's examples, but it may be also useful for other frameworks.
```